src/Symfony/Component/Translation/Command/XliffLintCommand.php

```diff
@@ -33,13 +33,15 @@ class XliffLintCommand extends Command
     private $displayCorrectFiles;
     private $directoryIteratorProvider;
     private $isReadableProvider;
+    private $requireStrictFileNames;
 
-    public function __construct(string $name = null, callable $directoryIteratorProvider = null, callable $isReadableProvider = null)
+    public function __construct(string $name = null, callable $directoryIteratorProvider = null, callable $isReadableProvider = null, bool $requireStrictFileNames = false)
```

Comment:
Shouldn't we keep using `true` as default, to keep the same behavior than today by default ?